.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: release
+
+# run only on tags
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write # needed to write releases
+  id-token: write # needed for keyless signing
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
```

Comment:
I find these comments fall out of date quickly because dependabot only updates the SHA and not the comment 😢 

Can you remove the comment parts?